### PR TITLE
Add standard thoughtbot documents

### PR DESCRIPTION
**Pull request by thorncp**
* Convert README to markdown
* Add LICENSE
* Add CONTRIBUTING
* Lint Markdown
* Lint itself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+We expect all contributors to follow thoughtbot's [code of conduct].
+
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/hound-python/graphs/contributors
+[code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
+
+## License
+
+hound-python is Copyright (c) 2015 thoughtbot, inc. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
```
Comment (teoljungberg) [inline README.md]: How does the link to LICENSE differ from Flake8 abouve? Should we write `[LICENSE][]` as done above?

Comment (thorncp) [inline README.md]: Maybe it's a GitHub-ism? I dunno, this was mostly copied from the Go service.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Hound-Python
+
+[![Build Status](https://circleci.com/gh/thoughtbot/hound-python/tree/master.svg?style=svg)](https://circleci.com/gh/thoughtbot/hound-python/tree/master)
+
+Python review service for Hound. Backed by [Flake8][].
```
Comment (gylaz) [inline README.md]: Is the `[]` necessary here?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+[CONTRIBUTING]: CONTRIBUTING.md
+[contributors]: https://github.com/thoughtbot/hound-python/graphs/contributors
+[code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
+
+## License
+
+hound-python is Copyright (c) 2015 thoughtbot, inc. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
```
Comment (gylaz) [inline README.md]: Is the leading slash necessary? I've seen both, absolute and relative, paths. Should we pick one and stick to it?

File: README.md (hunk 2 above)
Comment (thorncp) [inline README.md]: Nope, removed in 835741cf21a026ef060a0c23c90971877c8f1f9f

**Pull request merged**